packages/ui/useScroll/useScroll.js

```diff
@@ -392,6 +392,9 @@ const useScrollBase = (props) => {
 		}
 
 		return () => {
+			if (ref.scrollStopJob.alive()) {
+				ref.scrollStopJob.run();
+			}
```

Comment:
There is another scrollStopJob.stop() call in the following code. If this code works first, then the `ref.scrollStopJob.alive()` would be false and it works incorrectly. Am I right?

```
	useEffect(() => {
		mutableRef.current.resizeRegistry.parent = context;

		// componentWillUnmount
		return () => {
			const {... scrollStopJob} = mutableRef.current;

			...
			scrollStopJob.stop();
```

Reply:
I got it! Their lifecycles are different.